libs/index.md

```diff
@@ -70,6 +70,10 @@ Advanced usage (for badge makers):
       ['latest stable release', '/github/release/babel/babel/stable'],
       ['latest tag', '/github/tag/micromatch/micromatch'],
       ['watchers', '/github/watchers/micromatch/micromatch'],
+      ['status', '/github/status/micromatch/micromatch'],
+      ['status', '/github/status/tunnckoCore/badgen-service/github-status'],
+      ['status branch', '/github/status/micromatch/micromatch/dev'],
```

Comment:
micromatch doesn't have a `dev` branch

Reply:
It have.. :D

Reply:
Isn't the dev branch this https://github.com/micromatch/micromatch/tree/dev ?